### ISSUE TITLE
Multiple indexers find no results - conflict between Cloudflare and use of VPN/proxy in Jackett

### ISSUE THREAD

**Issue by ilike2burnthing** [Jackett/Jackett#7325]
Latest Docker v0.13.201-ls55, though it has happened over the last few days.

Enhanced log attached.
[log.txt](https://github.com/Jackett/Jackett/files/4251302/log.txt)

Tested by using the site itself, with and without VPN, no issues there.

Cheers!

**Comment by garfield69** [comment]
I just tested this and it works for me.
Albeit it is taking nearly 15 seconds to respond (including the mandatory 5 seconds wait from Cloudflare). Using the browser directly takes just as long.

have you tried again since you reported the issue?

It may well have been a genuine error on their server, I see `Returning ServiceUnavailable` in your log. Maybe the site server was under pressure and did not respond in time and timed out.

**Comment by ilike2burnthing** [comment]
Yeah, tried it several times a day since it stopped returning results, never any change. Tried it multiple times there now, same result. Also tried it without the proxy enabled in Jackett, still no results. All attempts appear to take between 15-20s (in line with what the site itself takes).

Is there any more information I can give you?

**Comment by garfield69** [comment]
no I don't think there is anything more you can provide. Since it works for me I can rule out Jackett.
this leaves your platform, your network, your ISP provider, basically everything out of Jackett's control.
@ngosang  @cadatoiva  @junglebus  any of you having problems with this indexer?

**Comment by ilike2burnthing** [comment]
So I started up a sandbox and ran a fresh copy of the Windows binaries for Jackett in it, Badass worked fine. The only setting that caused an issue was when I added the NordVPN server as a proxy - found no results.

I removed the proxy just to double-check, applied and let it refresh, tested again, but still no results. It wasn't until I shutdown the tray program and restarted it that results were found.

To test this in my actual Docker Jackett install I removed the proxy, applied, let it refresh, tested, and found no results. Applying server settings multiple times didn't help. So I restarted the container via Portainer, and only then did Badass work.

Whatever is happening during that refresh isn't actually removing the proxy settings. I've confirmed this by successfully running tests on indexers which should fail without the proxy, both in the Docker container and the sandboxed Windows install. 

Aside from that whole new issue, using the exact same NordVPN server in a browser works fine for Badass, so I'm not sure why it's causing a problem in Jackett.

**Comment by garfield69** [comment]
I've no experience in using proxies / VPN so I'll invite the other devs to contribute.

**Comment by cadatoiva** [comment]
I'll look through the code to see why the proxy settings aren't updating until restart of the application. In the meantime can we get the enhanced logs for when you search this tracker with the proxy enabled?

**Comment by ilike2burnthing** [comment]
Sorry for the confusion, the attached log in the first post is with the proxy enabled.

**Comment by ilike2burnthing** [comment]
v0.13.211-ls55 - same issue

**Comment by cadatoiva** [comment]
In your logs, it's showing that cloudflare DDoS protection page is preventing Jackett from reaching the site to run a search, which you can see at these lines:
```
2020-02-25 17:15:02.8742 Debug Executed controller factory for controller Jackett.Server.Controllers.IndexerApiController (jackett) 
2020-02-25 17:15:02.8756 Debug WebClient(HttpWebClient2NetCore).GetString(Url:https://badasstorrents.com/torrents/search/) 
2020-02-25 17:15:58.7563 Debug WebClient(HttpWebClient2NetCore): Returning ServiceUnavailable =>
```
Putting the HTML from the log into a browser creates the following screenshot:
![image](https://user-images.githubusercontent.com/2086652/75407268-309d0b00-58d8-11ea-9730-1ad1d5d78821.png)

Looks like BAT is a YAML based indexer, and if I'm not mistaken @garfield69 those don't have CloudFlare bypass? If that's true, we'll need to rewrite this one in C# to fix the main issue @ilike2burnthing is experiencing

**Comment by garfield69** [comment]
>Looks like BAT is a YAML based indexer, and if I'm not mistaken those don't have CloudFlare bypass? 

@cadatoiva  not sure I understand what you mean by ** CLoudflare bypass**. The Cardigann processor http(s) requests all goes through whichever of the 4 httpWebClient* module is active, and they in turn all invoke the cloudflareSolverRe... or do you mean something else?

**Comment by cadatoiva** [comment]
That's what I was referring to.

**Comment by ngosang** [comment]
This is caused by a new Cloudflare challenge that affect many sites => https://github.com/RyuzakiH/CloudflareSolverRe/issues/14
Affected:
* badasstorrents
* extratorrent-cd
* bteye
* limetorrents

Related: #7210

**Comment by cadatoiva** [comment]
Closing this issue in favor of centralized tracking in #7397

**Comment by ilike2burnthing** [comment]
Should I open a new ticket for the issue of proxy settings not being removed on refresh?

**Comment by cadatoiva** [comment]
If you could test it on a couple of trackers to make sure it's not working, or if it's just this one. When I reviewed the code there was no reason why it shouldn't be working.

**Comment by ilike2burnthing** [comment]
"I've confirmed this by successfully running tests on indexers which should fail without the proxy, both in the Docker container and the sandboxed Windows install."

Done

**Comment by cadatoiva** [comment]
Then please open a new issue for the proxy not refreshing as a core Jackett function.

**Comment by ilike2burnthing** [comment]
Thought I would try again since the fix by @ngosang is up and running now - https://github.com/Jackett/Jackett/pull/7546

Unfortunately both Badass and the LimeTorrents alts mentioned here - https://github.com/Jackett/Jackett/issues/7496 (.asia .co and .pro) all fail when using the NordVPN proxy.

Using the exact same server in browser works fine for those sites, but not in Jackett, which leads me to think it's not just a simple IP blacklist.

Looking at the log (admittedly as a complete novice), it is showing the same issue as before, just with some differences in formatting - [enhanced log.txt](https://github.com/Jackett/Jackett/files/4308147/log.txt)

```
Debug WebClient(HttpWebClient2NetCore).GetString(Url:https://badasstorrents.com/torrents/search/)
Debug WebClient(HttpWebClient2NetCore): Returning ServiceUnavailable

[...HTML...]

Info Found 0 releases from Badass Torrents
```

Same issue in a clean Windows Sandbox install using the NordVPN proxy.

Any help would be appreciated. Thanks!

**Comment by garfield69** [comment]
The BadassTorrents and LimeTorrents indexers on my win10 Jackett 0.13.560 without VPN work perfectly fine, so the issue is definitely with your vpn and I doubt very much there is anything Jackett can do about this.
But I will re-open and see if the c# devs concur.

**Comment by ilike2burnthing** [comment]
My only hesitancy in agreeing is that those sites work in browser using that same NordVPN server. I get the Cloudflare anti-DDoS wait page, and after 15s or so they load fine and I'm able to search and download without issue. It's a head scratcher :/

**Comment by garfield69** [comment]
please confirm the version of Jackett you are on, by opening your Jackett dashboard and scrolling to the bottom, and reporting it here.
the last post had you on 0.13.211, are you now on  v0.13.529 - v0.13.560 ?

**Comment by ilike2burnthing** [comment]
Oh sorry, completely forgot - v0.13.560, for both the Docker image and the Windows sandbox test.

**Comment by ilike2burnthing** [comment]
Wasn't working yesterday on 529 either.

**Comment by ngosang** [comment]
> The BadassTorrents and LimeTorrents indexers on my win10 Jackett 0.13.560 without VPN work 

Same from here. This is not related to Cloudflare, must be your proxy. Limetorrents has several domains, try with other.

**Comment by ilike2burnthing** [comment]
Oh I'm fine using a different domain for LT (.info and .com don't use the Cloudflare anti-DDoS protection, thus don't have this issue), I was just using it as the only other indexer being affected the same way as BT (that I'm aware of anyway).

Are either of you, or anyone else, able to use a proxy and access those sites with Jackett?

If so, that would confirm the issue being either specific to NordVPN in Jackett (despite it working in browser), or something to do with my local setup. If not, then it might be some conflict between the general use of a VPN/proxy, Cloudflare, and Jackett.

**Comment by garfield69** [comment]
@cadatoiva  Do you use a proxy/vpn?

**Comment by cadatoiva** [comment]
I have a proxy/vpn I can use for testing, but I do not proxy/vpn Jackett under normal circumstances. I'll look at this one in a couple hours after I wake up.

**Comment by ilike2burnthing** [comment]
Badass has dropped the Cloudflare DDoS protection today, so now it's just LimeTorrents .asia .co and .pro available for testing purposes (at least of the ones I'm using).

**Comment by ilike2burnthing** [comment]
Just realised that ExtraTorrent also has the same issue - 
[log - ExtraTorrent.txt](https://github.com/Jackett/Jackett/files/4330329/log.-.ExtraTorrent.txt)

**Comment by cadatoiva** [comment]
Is this still an issue in the latest version?

**Comment by ilike2burnthing** [comment]
I'm on 0.14.136.0, issue is still present for LT .asia .co and .pro as well as ET.

**Comment by ilike2burnthing** [comment]
Tested in v0.14.222-ls58, but added by v0.14.164-ls58, LT .unblockit.pro proxy also failing with the same issue

**Comment by ilike2burnthing** [comment]
@cadatoiva are you able to use your VPN/proxy in Jackett to test?

**Comment by cadatoiva** [comment]
Jackett is getting the following page when browsing via proxy on one of the affected sites:

Please enable cookies.
Error 1020 Ray ID: 57821d014eb4e750 • 2020-03-22 18:53:03 UTC
Access denied
What happened?
This website is using a security service to protect itself from online attacks.

Cloudflare Ray ID: 57821d014eb4e750 • Your IP: xxx.xxx.xxx.xxx • Performance & security by Cloudflare

**Comment by cadatoiva** [comment]
Further research shows it's a straight IP ban, because using the same proxy on my browser leads to the same page. Summarily blocked. Not a Jackett issue, can't fix.

**Comment by ilike2burnthing** [comment]
> Using the exact same server in browser works fine for those sites, but not in Jackett, which leads me to think it's not just a simple IP blacklist. Also tried a few other servers, same issue.

```
2020-03-22 21:55:40.4663 Debug WebClient(HttpWebClient2NetCore).GetString(Url:https://extratorrent.si/search/?search=) 
2020-03-22 21:55:41.0827 Debug WebClient(HttpWebClient2NetCore): Returning ServiceUnavailable => <!DOCTYPE HTML>
<html lang="en-US">
<head>
  <meta charset="UTF-8" />
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1" />
  <meta name="robots" content="noindex, nofollow" />
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
  <meta http-equiv="refresh" content="12">
  <title>Just a moment...</title>
  <style type="text/css">
    html, body {width: 100%; height: 100%; margin: 0; padding: 0;}
    body {background-color: #ffffff; color: #000000; font-family:-apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, "Helvetica Neue",Arial, sans-serif; font-size: 16px; line-height: 1.7em;-webkit-font-smoothing: antialiased;}
    h1 { text-align: center; font-weight:700; margin: 16px 0; font-size: 32px; color:#000000; line-height: 1.25;}
    p {font-size: 20px; font-weight: 400; margin: 8px 0;}
    p, .attribution, {text-align: center;}
    #spinner {margin: 0 auto 30px auto; display: block;}
    .attribution {margin-top: 32px;}
    @keyframes fader     { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} }
    @-webkit-keyframes fader { 0% {opacity: 0.2;} 50% {opacity: 1.0;} 100% {opacity: 0.2;} }
    #cf-bubbles > .bubbles { animation: fader 1.6s infinite;}
    #cf-bubbles > .bubbles:nth-child(2) { animation-delay: .2s;}
    #cf-bubbles > .bubbles:nth-child(3) { animation-delay: .4s;}
    .bubbles { background-color: #f58220; width:20px; height: 20px; margin:2px; border-radius:100%; display:inline-block; }
    a { color: #2c7cb0; text-decoration: none; -moz-transition: color 0.15s ease; -o-transition: color 0.15s ease; -webkit-transition: color 0.15s ease; transition: color 0.15s ease; }
    a:hover{color: #f4a15d}
    .attribution{font-size: 16px; line-height: 1.5;}
    .ray_id{display: block; margin-top: 8px;}
  </style>

    <script type="text/javascript">
  //<![CDATA[
  (function(){
    var a = function() {try{return !!window.addEventListener} catch(e) {return !1} },
    b = function(b, c) {a() ? document.addEventListener("DOMContentLoaded", b, c) : document.attachEvent("onreadystatechange", b)};
    b(function(){
      
      var a = document.getElementById('cf-content');a.style.display = 'block';
      var isIE = /(MSIE|Trident\/|Edge\/)/i.test(window.navigator.userAgent);
      var trkjs = isIE ? new Image() : document.createElement('img');
      trkjs.setAttribute("src", "/cdn-cgi/images/trace/jschal/js/transparent.gif?ray=57832885bd0e3450");
      trkjs.id = "trk_jschal_js";
      trkjs.setAttribute("alt", "");
      document.body.appendChild(trkjs);
      
      setTimeout(function(){
        var s,t,o,p,b,r,e,a,k,i,n,g,f, uGpydkz={"jg":+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(+!![]))/+((!+[]+!![]+!![]+!![]+!![]+[])+(+[])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]))};
        g = String.fromCharCode;
        o = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
        e = function(s) {
          s += "==".slice(2 - (s.length & 3));
          var bm, r = "", r1, r2, i = 0;
          for (; i < s.length;) {
              bm = o.indexOf(s.charAt(i++)) << 18 | o.indexOf(s.charAt(i++)) << 12
                      | (r1 = o.indexOf(s.charAt(i++))) << 6 | (r2 = o.indexOf(s.charAt(i++)));
              r += r1 === 64 ? g(bm >> 16 & 255)
                      : r2 === 64 ? g(bm >> 16 & 255, bm >> 8 & 255)
                      : g(bm >> 16 & 255, bm >> 8 & 255, bm & 255);
          }
          return r;
        };
        t = document.createElement('div');
        t.innerHTML="<a href='/'>x</a>";
        t = t.firstChild.href;r = t.match(/https?:\/\//)[0];
        t = t.substr(r.length); t = t.substr(0,t.length-1); k = 'cf-dn-BLPtfFbgs';
        a = document.getElementById('jschl-answer');
        f = document.getElementById('challenge-form');
        ;uGpydkz.jg*=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((!+[]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![])+(!+[]+!![]+!![])+(+[])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]));uGpydkz.jg-=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((!+[]+!![]+!![]+!![]+[])+(+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]));uGpydkz.jg-=function(p){var p = eval(eval(e("ZG9jdW1l")+(undefined+"")[1]+(true+"")[0]+(+(+!+[]+[+!+[]]+(!![]+[])[!+[]+!+[]+!+[]]+[!+[]+!+[]]+[+[]])+[])[+!+[]]+g(103)+(true+"")[3]+(true+"")[0]+"Element"+g(66)+(NaN+[Infinity])[10]+"Id("+g(107)+")."+e("aW5uZXJIVE1M"))); return +(p)}();uGpydkz.jg-=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![])+(+[]));uGpydkz.jg-=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]));uGpydkz.jg-=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(+!![]))/+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(+[])+(!+[]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![])+(+!![]));uGpydkz.jg*=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((!+[]+!![]+[])+(!+[]+!![]+!![]+!![])+(!+[]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]));uGpydkz.jg*=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+!![]));uGpydkz.jg-=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![])+(+!![])+(+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+!![]))/+((+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]));uGpydkz.jg*=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]))/+((+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]));uGpydkz.jg*=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]))/(+(+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])))+(function(p){return eval((true+"")[0]+".ch"+(false+"")[1]+(true+"")[1]+Function("return escape")()(("")["italics"]())[2]+"o"+(undefined+"")[2]+(true+"")[3]+"A"+(true+"")[0]+"("+p+")")}(+((!+[]+!![]+!![]+!![]+[])))));uGpydkz.jg-=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![])+(+[])+(!+[]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![])+(!+[]+!![]+!![]))/+((+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+!![])+(+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]));uGpydkz.jg+=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![])+(!+[]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![]))/+((+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]));uGpydkz.jg*=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((!+[]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![])+(!+[]+!![]+!![]+!![])+(+!![])+(+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]));uGpydkz.jg-=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((!+[]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]));uGpydkz.jg+=+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![])+(+!![])+(+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+!![]))/+((!+[]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+!![])+(!+[]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]));a.value = (+uGpydkz.jg).toFixed(10); '; 121'
        f.action += location.hash;
        f.submit();
      }, 4000); /*eoc*/
    
    }, false);
  })();
  //]]>
</script>



</head>
<body>
  <table width="100%" height="100%" cellpadding="20">
    <tr>
      <td align="center" valign="middle">
          <div class="cf-browser-verification cf-im-under-attack">
  <noscript>
    <h1 data-translate="turn_on_js" style="color:#bd2426;">Please turn JavaScript on and reload the page.</h1>
  </noscript>
  <div id="cf-content" style="display:none">
    
    <div id="cf-bubbles">
      <div class="bubbles"></div>
      <div class="bubbles"></div>
      <div class="bubbles"></div>
    </div>
    <h1><span data-translate="checking_browser">Checking your browser before accessing</span> extratorrent.si.</h1>
    
    <p data-translate="process_is_automatic">This process is automatic. Your browser will redirect to your requested content shortly.</p>
    <p data-translate="allow_5_secs">Please allow up to 5 seconds&hellip;</p>
  </div>
   <a href="https://tempestsw.com/nightingalesguilt.php?c=0"></a>
  <form id="challenge-form" action="/search/?search=&amp;__cf_chl_jschl_tk__=ad1786f3399e4dadd8bd3f02adfdd2fe2324ea09-1584914141-0-AZIJz61SWN0rsvdz7ej9N2yLfx6rW1VZhULlhBo37CLjRI75j00MCMUp_Bce4ps6XLZ7cZ3HCEaIB9Hf3NaWZ5E-7jj1ckCla7Gci0vCdZClVJlGU6OIVt9f-i3MWiamwu88YIazuvRfoBO0O0tMDJLka4s-qk6M6ctlnAYUepoBw97fFaq4chbstLCrErz6eNIdYuEyb5MBoNEhsbMrmJ9m6Bl6_mKsMZTTzt2V1i6MVg99p8wLCcAFRVh0AHHoA9j7eBcyzKJ8uma237Pcs1IZAT4xEjbu4ey6r9qnb7ev" method="POST" enctype="application/x-www-form-urlencoded">
    <input type="hidden" name="r" value="ae6b411e3a0f87878fa9e82fed71e7f0afdbcc91-1584914141-0-AQfAKOOOc+Cpo3Zdafx68pNNGo3mqurOteCzb0SVCkFbfXBhXkOwNgjxffselHYGB2a2TAt9LGnn9c+v6Faow8NUiayUtSWEsOBACGNRr/zkYi6Ntf8pVs7kvjITR+lyz/MYF6TZ4G2Uj0hqKrRLuKHneszleB07c1SIIFWXMpViB9oJjBxeRHvQdZCUHCzcWWLwuD7a8eZXjS1dcUbN6vvZjEy3Uj7B1Hmm/mAcSChdLF+3Ktm9n6nJyIOpi7pKGV1hphgcpqWIPaPQ2PP89UD3yZ0OnaHzpCaY7927VFoGHHwg7mu7piq0AdKIcgwDGSud013nAfLZ/xXORWceX7qMJnwVbildv5k2vFdDLIN+vIJ/iZGKsRk1SKwjSXc84XkiEaSHHaVLESxlxAUbCmCiJHQcXt6vck61MJD2PeKdg5d+bh+rxVE6ySaU2ZX+wIm8iLwSZYQrz/5mHeYY6KxcdHSfYcujGxJJBwvByuAL6FFC3DV0YM86+L4TOntIh+kpVmm1EM0McmQRipdtYBDv1W+QULy3AWL8oYM1ZTdwMQGgmpsr9GxDMDCFO9mveDIjXIfKmOzDnH5dInogC3dqLcx4zod/EoVSUECdlI2IZCGzCc1xo/MnrpWxnxddKsjKzF24TWtY7/oxcBKaJh37frrEx5/gqrXWoJipa5qyXXn2Rv/zV57XIoQXpxAZbz/a+kViexYkljHhf/h+Wp67GCX2gaWCzWhuA2DGm8SJcWPl/mdkMO6jixGKRKMSZvL5X6tZl+4CJArGbR0e5hJ92XjObci0cK1OVHjIecSCVdjcgeQ0JTrzXQAyxRgwgTjwfaubU0doHZvsyxwABGA3a35syUal2Db3/nR6PguxQNOIJzmhewh5z/aeWIO5K10vfK5jRh0QTTU2KSlCvZdO4u1+79cEw3YBf5iD+pmrUcyl6seczb7zjgWrqaLcSlO2VajO8JC1ziz2MZwZMTE/QpXuvSRiDZ+1dqPpYqGLFk3s0T97oXrrsFZRd3CbmprOYluw9U9ngITmeGZ9Yglcq/leyf8QOL3LkewSfSOWe3dk9V5vvT5nQY9xSylRccDXXI4C1bP+QMPzw3ZYqKNlb3EHy1WrsyhDI7WTjTe/yzSblRUqq0xfUubE8nOm2cXIAB/3zxW4kxTBWvbFQ8Dlo7/c9gtBE3dfI04cqTksJFG2Ex2pl6Rjviez2+uvoCbixcO+27EL9in8j2dlgtD7LNTWj4xVNlT71G1zRSfTY7ILItW3HICweb1doKxXFq4hAy8NLuwOyXAqYUki5PgjaEX5pgbc99331hqYgDVKR3vUXXh/XaI72LD01sLwDIz4M0WQwZXC+JoKFB190HquwKCOWQCJRlqwB9gHsihLjk9eLfVx7BSr23F+wwHayewhLM2mj0ZpO+53Vhp54DqAIq3P0PG0Mr5+Q+YKj9Y77mm55Qu8SBP7WOGLWSzOgytr+G2SW5PQjRWakbMOjM0="/>
    <input type="hidden" id="jschl-vc" name="jschl_vc" value="437ae44ee10f023fc7553470927d6b00"/>
    <input type="hidden" name="pass" value="1584914145.08-PCrH7Fzxpk"/>
    <input type="hidden" id="jschl-answer" name="jschl_answer"/>
  </form>
  
  <div style="display:none;visibility:hidden;" id="cf-dn-BLPtfFbgs">+((!+[]+!![]+!![]+!![]+!![]+!![]+!![]+[])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))/+((+!![]+[])+(!+[]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]+!![])+(+[])+(!+[]+!![]+!![])+(!+[]+!![]+!![]+!![]+!![]+!![]+!![]))</div>
  
  <div id="trk_jschal_nojs" style="background-image:url('/cdn-cgi/images/trace/jschal/nojs/transparent.gif?ray=57832885bd0e3450')"> </div>
</div>

          
          <div class="attribution">
            DDoS protection by <a href="https://www.cloudflare.com/5xx-error-landing?utm_source=iuam" target="_blank">Cloudflare</a>
            <br />
            <span class="ray_id">Ray ID: <code>57832885bd0e3450</code></span>
          </div>
      </td>
     
    </tr>
  </table>
</body>
</html>
 
2020-03-22 21:55:41.1377 Info Found 0 releases from ExtraTorrent.cd 
2020-03-22 21:55:41.1490 Info Executed action Jackett.Server.Controllers.IndexerApiController.Test (jackett) in 686.1648ms 
2020-03-22 21:55:41.1516 Info Executed endpoint 'Jackett.Server.Controllers.IndexerApiController.Test (jackett)' 
2020-03-22 21:55:41.1549 Error System.Exception: Found no results while trying to browse this tracker
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/vsts/work/1/s/src/Jackett.Common/Services/IndexerManagerService.cs:line 225
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 144
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 70 System.Exception: Found no results while trying to browse this tracker
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in /home/vsts/work/1/s/src/Jackett.Common/Services/IndexerManagerService.cs:line 225
   at Jackett.Server.Controllers.IndexerApiController.Test() in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 144
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 70
```

It loads up the DDoS protection page and then doesn't continue any further, it doesn't deny access due to an IP block.

I just set up another test:
- clean Windows Sandbox
- fresh install of Jackett (no proxy settings this time)
- setup NordVPN in Windows Sandbox
- use same server as always
- connect to server
- test ET and affected LT domains

All tests are successful.

As far as I can tell I've ruled out IP block, my local setup, my Jackett setup, my VPN, and my ISP.
The issue is specific to using an otherwise working VPN/proxy within Jackett for those sites.

So can this issue please be reopened? Again... lol 😄

**Comment by cadatoiva** [comment]
Ok, I know we did most of these, but please test the following scenarios and verify 

- Other sites working while Jackett is on the proxy
  - Including other sites that use CloudFlare
- These sites work while not on the proxy, and they've triggered CloudFlare during the clear test
- With the proxy unset, route all traffic through a VPN to the same server, does this work?

**Comment by ilike2burnthing** [comment]
Current list of indexers:
```
1337x 7torrents BadassTorrents BTDB BTSOW Demonoid ETTV EXTTorrents
ExtraTorrent.cd EZTV GloDLS IBit Idope Isohunt2 KickAssTorrent LimeTorrents
MagnetDL Monova Newstudio RARBG RuTor ShowRSS SkyTorrentsClone SolidTorrents
ThePirateBay TopNow Torlock TOROS TorrentDownloads TorrentDownload
TorrentFunk TorrentGalaxy.org TorrentParadise TorrentProject2 TorrentQuest
TorrentRex YourBittorrent Zooqle
```
Only Demonoid, ExtraTorrent.cd, and TorrentRex currently fail. All others have no issues - katcr.co is regularly down so I use katcr.black-mirror.xyz as a working proxy

Demonoid has its own issue and will hopefully be resolved soon - https://github.com/Jackett/Jackett/pull/7771

TorrentRex's site is constantly up and down, so that can be ignored.

ExtraTorrent is the only one displaying this issue.

However, if you then switch LimeTorrent to one of the 4 affected alternate URLs, then it displays the same issue. Only .info and limetor.pro are fine, and they're the only ones which don't use Cloudflare's DDoS protection).

I had a look at some other alt sites:
- only 1337x's .to domain is successful, all others fail - all use Cloudflare's DDoS protection, whereas .to does not
- Demonoid unblockit.pro proxy shows this issue instead of the `Cloudflare reCaptcha [...] aggressive DDoS mitigation mode` which the .is site shows
- BTDB, ETTV, EZTV, KAT, MagnetDL, Monova, Torlock, TorrentDownload, TorrentFunk, TorrentGalaxy, and Zooqle unblockit.pro proxies fail - all use Cloudflare DDoS protection

All those sites work in browser when using the same NordVPN server.

With no Jackett proxy settings, all unblockit.pro proxies are successful (including 1337x) except Demonoid, which also fails in browser, seemingly due to the aforementioned 'aggressive DDoS mitigation mode' also affecting the proxy.

With no Jackett proxy settings, 1337x.to and LimeTorrent.info fail, as well as all proxies (except for their own unblockit.pro proxy), which all fail in browser due to ISP/regional DNS blocks.

Hope that helps!

**Comment by ilike2burnthing** [comment]
A few extra tests of sites I don't use, 2 with Cloudflare, 1 with a different DDoS protection provider.

BTeye & Torrentz2 (Cloudflare DDoS Protection):
- fail in Jackett with proxy settings
- succeed in Jackett without proxy settings
- succeed in browser with same NordVPN server

Anidex ([DDoS Guard](https://ddos-guard.net/)) succeeds in all cases. No issues.

**Comment by cadatoiva** [comment]
Is the result the same when NordVPN client is being used to redirect Jackett without proxy? That's the one test I'm most interested in finding out. So basically set up as follows

- Proxy settings in jacket are clear (no proxy)
- On the test OS, NordVPN client is installed and directing traffic
- Verify what kind of functionality Jackett has.

**Comment by ilike2burnthing** [comment]
> I just set up another test:
> - clean Windows Sandbox
> - fresh install of Jackett (no proxy settings this time)
> - setup NordVPN in Windows Sandbox
> - use same server as always
> - connect to server
> - test ET and affected LT domains
> 
> All tests are successful.

End of my second to last long comment. Easy to miss with that wall of text.

 So yea, proxy used within Jackett, tests for those indexers fail. VPN used outside of Jackett (same server, no proxy settings in Jackett), tests for those indexers are successful.

**Comment by cadatoiva** [comment]
Well, isn't that strange indeed. Good thing I'm in the middle of a major refactor of the entire section of code this pertains to. I'll keep a close eye on the documentation while completing the work.

One more thing I'd like to check on though, can you add `--UseClient httpclient` to your command line and test again?

**Comment by ilike2burnthing** [comment]
`--UseClient httpclient` results in no change in the Windows sandbox install, using either Jackett proxy settings and no VPN, or VPN and no Jackett proxy settings. Throwing it into RUN_OPTS for Docker gives the same result (or rather, lack thereof).

**Comment by cadatoiva** [comment]
Ok, so it should affect all versions of the current implementation of the download client. Can you test this version from the alpha builds of the feature change? No need to try `--UseClient`` 

https://dev.azure.com/jackett/jackett/_build/results?buildId=124&view=artifacts&type=publishedArtifacts

**Comment by ilike2burnthing** [comment]
Same results. Got my hopes up at one point until I realised it was just returning the cached results.

**Comment by cadatoiva** [comment]
Thanks, then the changes so far into the refactor haven't tracked down this issue.

**Comment by ilike2burnthing** [comment]
Tested all the public indexers, just their primary URL as there's too many to try every proxy, if they have any. So here's a complete list of those currently affected by this issue:
- BTeye
- cpasbien
- divxtotal
- ExtraTorrent
- 1337x - .gd .is .st .ws .eu .se
- LimeTorrent - .asia .co limetor.pro
- all unblockit.pro proxies

All of those use Cloudflare DDoS Protection*, none of the others seem to. All work in browser with same NordVPN server. Those not blocked for me by ISP work in Jackett without proxy.

Hopefully that gives you a bit more to work with.

* DivxTotal appears to be using a skinned Cloudflare page from looking at the html

**Comment by ilike2burnthing** [comment]
ExtraTorrent is currently working with the proxy enabled.

I'll update if that changes back, otherwise assume it's no longer part of the issue.

**Comment by Jerrk** [comment]
**KickAssTorrent (kat.li)** and **Torrentz2** does not work with my proxy activated, but they do work with it disabled.

**Comment by ilike2burnthing** [comment]
What site are you using for kat.li? kickass.ws is working for me. All unblockit.one proxies fail, so that could be why. But yes, Torrentz2 fails here too.

**Comment by Jerrk** [comment]
both https://kickass.ws/ and https://kickass.unblockit.one/ fail to connect and show the same error of no results

**Comment by ilike2burnthing** [comment]
Can you provide enhanced logs for kickass.ws? What isohunt2 and torrentz2 links are you using?

**Comment by Jerrk** [comment]
torrentz2 - https://torrentz2.uk-unblock.xyz/
isohunt2 - https://isohunt2.net/ https://isohunt.nz/ neither works

heres enhanced log. i first run the test with proxy disabled and it works, i then turn on my proxy and it fails
[log.txt](https://github.com/Jackett/Jackett/files/4538508/log.txt)

**Comment by ilike2burnthing** [comment]
Using the same proxy details, can you access those sites in browser? Jackett is receiving a 403 Forbidden page.

**Comment by cadatoiva** [comment]
Reminder that currently Proxy support has been downgraded to experimental in Jackett and is not guaranteed to work correctly. Unless you need a proxy/VPN to bypass geo-blocking or ISP content restrictions, it is recommended to not use a proxy. If a proxy/VPN is needed, then using an external VPN is the preferred method. 

This holds true until I finish rebuilding the entire web downloader system.

**Comment by ilike2burnthing** [comment]
Issue no longer relevant due to FlareSolverr. Only thing holding up use of Jackett's proxy settings with FlareSolverr is https://github.com/FlareSolverr/FlareSolverr/issues/26
